theano/sandbox/cuda/basic_ops.py

```diff
@@ -575,14 +577,6 @@ def __init__(self, reduce_mask, scalar_op, pre_scalar_op=None):
         if pre_scalar_op:
             assert pre_scalar_op.nin == 1
 
-    def __str__(self):
-        pre = ""
-        if self.pre_scalar_op:
-            pre = "pre=%s,red=" % str(self.pre_scalar_op)
-        return "GpuCAReduce{%s%s}{%s}" % (
-            pre, str(self.scalar_op),
-            ','.join(str(i) for i in self.reduce_mask))
```

Comment:
I think you commented on a particular commit. this and one more comment seem to have been addressed at the top of this branch